collect_app/src/main/java/org/odk/collect/android/activities/FormChooserList.java

```diff
@@ -172,9 +158,10 @@ protected void onStop() {
 
     @Override
     public void syncComplete(String result) {
-        Timber.i("Disk sync task complete");
-        TextView tv = findViewById(R.id.status_text);
-        tv.setText(result.trim());
+        Timber.i("Disk scan complete");
```

Comment:
This block of code is duplicated four times. Can we eliminate? Here’s one solution:

https://github.com/dcbriccetti/collect/commit/3267fc942217dcee1501cf7cf95c15b5fdae618e

Reply:
After our discussion in the commit in my branch above, I’m content to leave this alone.

Reply:
@dcbriccetti I've annotated the parameter as `@NonNull` and moved the empty string check to the util class